apps/client/ui/NavMenu/NavMenu.tsx

```diff
@@ -48,21 +50,25 @@ export const NavMenu: FC<NavMenuProps> = ({
         <li
           key={item}
           className={clsx(
-            'relative text-lg font-bold capitalize cursor-default whitespace-nowrap',
+            'h-12 leading-[48px] relative text-lg font-bold capitalize cursor-default whitespace-nowrap',
             {
               'hover:text-text-color transition-colors transition-ease-in-out duration-100': !activeItem,
               'text-text-color cursor-pointer': activeItem === item,
               'text-text-color/[.50]': activeItem !== item,
             },
           )}
-          onClick={() => onClick(item)}
+          onMouseOver={() => onMouseOver(item)}
+          onMouseLeave={onMouseLeave}
         >
           {item}
           {activeItem === item && (
             <DropdownMenu
-              className="absolute top-10 text-sm font-semibold"
+              className="absolute text-sm font-semibold"
               menuItems={dropdownMenuItems}
+              //it doesn't work without these two function here, as you showed in your example
```

Comment:
right 😄 it looks really similar to what I did yesterday but dropdown hides without these hooks - don't know what's the difference 😄